Models/Computers.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BangazonAPI.Models
+{
+    public class Computers
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
+        public DateTime PurchaseDate { get; set; }
+
+        [Required]
```

Comment:
If the decomission date is required, does that mean there are no computers that are not decomissioned? Which means no operating computers???

Reply:
A "?" after DateTime would fix that right? To show that it can be null?